src/components/ui/select/SelectValue.vue

```
@@ -1,12 +1,13 @@
 <script setup lang="ts">
 import type { SelectValueProps } from 'reka-ui'
-import { SelectValue } from 'reka-ui'
+import { SelectValue, useForwardProps } from 'reka-ui'
```

ESLint

[error] Unable to resolve path to module 'reka-ui'.

(import-x/no-unresolved)